sdk/attestation/azure-security-attestation/README.md

```diff
@@ -170,9 +168,9 @@ InitTime data refers to data which is used to configure the SGX enclave being at
 Creates an instance of the Attestation Client at uri `endpoint`.
 
 ```python
-        attest_client = AttestationClient(
-            credential=DefaultAzureCredential(),
-            instance_url=base_uri)
+attest_client = AttestationClient(
+    credential=DefaultAzureCredential(), 
+    endpoint=base_uri)
```

Comment:
Normally we prefer endpoint first, then credential.

Reply:
This was made throughotu the service yesterday, I missed the snippet in the readme (it's annoying that JS and Python don't have a mechanism to automatically sync snippets in readme/changelog files :().